lib/Form/FormInput.js

```diff
@@ -50,26 +54,35 @@ class FormInput extends Component {
       value,
       fullWidth,
       paddingSmall,
+      hasError,
+      errorMessage,
       ...rest
     } = this.props;
     const inputValue = this.state.value;
 
     const classNames = cx(`form__input ${className}`, {
       'form__input--noborder': noBorder,
       'form__input--full-width': fullWidth,
-      'form__input--padding-small': paddingSmall
+      'form__input--padding-small': paddingSmall,
+      'form__input--has-error': hasError,
+      'form__input--display-error': hasError && errorMessage
     });
 
     return (
-      <input
-        type={type}
-        value={inputValue}
-        onChange={this.handleOnChange}
-        className={classNames}
-        placeholder={placeholder}
-        autoFocus={this.props.focusOnMount} // eslint-disable-line jsx-a11y/no-autofocus
-        {...rest}
-      />
+      <Fragment>
+        <input
+          type={type}
+          value={inputValue}
+          onChange={this.handleOnChange}
+          className={classNames}
+          placeholder={placeholder}
+          autoFocus={this.props.focusOnMount} // eslint-disable-line jsx-a11y/no-autofocus
+          {...rest}
```

Comment:
When a screen reader tabs through this input, it has no way of knowing whether it is valid or not, an easy fix to this is to use the attribute `aria-invalid` and passing it true or false depending on whether it is valid